routes.go

```diff
@@ -74,7 +75,7 @@ func ExtractInteraction(r *http.Request) (*slack.InteractionCallback, error) {
 	}
 	var interaction slack.InteractionCallback
 	if err = json.Unmarshal([]byte(payload[0]), &interaction); err != nil {
-		return nil, errors.New("unable to unmarshal interaction")
+		return nil, errors.New(fmt.Sprintf("unable to unmarshal interaction: %s", err))
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to unmarshal interaction: %w", err))
```
This is a great fit for `fmt.Errorf` and enables native error wrapping.

Reply:
I had a feeling there was something I was missing along those lines. Thanks!

Reply:
Thank _you_!